Content.Client/GameObjects/Components/Disposal/DisposalMailingUnitWindow.cs

```diff
@@ -99,7 +99,7 @@ public DisposalMailingUnitWindow()
                                 HorizontalExpand = true,
                                 Children =
                                 {
-                                    new Label {Text = Loc.GetString("Eject:")},
+                                    new Label {Text = Loc.GetString("disposal-mailing-unit-eject-label")},
```

Comment:
Just want to point out, related to previous conversation, it's very good that this *isn't* the generic entry, given that it's got added punctuation.

So just a little praise here :P 👍

Reply:
yeah, I think I missed a few places where more specific tokens would be better, but I'll fix this once I parse the code again